src/components/application_manager/test/commands/mobile/alert_request_test.cc

```diff
@@ -63,10 +66,55 @@ using am::commands::AlertRequest;
 using am::commands::MessageSharedPtr;
 using am::event_engine::Event;
 using am::MockMessageHelper;
+using policy_test::MockPolicyHandlerInterface;
 
 typedef SharedPtr<AlertRequest> CommandPtr;
+namespace {
+const uint32_t kDefaultTimeout = 1000u;
+const uint32_t kConnectionKey = 1u;
+const uint32_t kCorrelationId = 2u;
+const uint32_t kFunctionId = 3u;
+}  // namespace
 
 class AlertRequestTest : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  AlertRequestTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock())
+      , mock_app_(CreateMockApp())
+      , msg_(CreateMessage(smart_objects::SmartType_Map)) {}
+
+  void SetUp() OVERRIDE {
+    (*msg_)[am::strings::params][am::strings::function_id] = kFunctionId;
+    (*msg_)[am::strings::params][am::strings::connection_key] = kConnectionKey;
+    (*msg_)[am::strings::params][am::strings::correlation_id] = kCorrelationId;
+
+    ON_CALL(mock_app_manager_, application(kConnectionKey))
+        .WillByDefault(Return(mock_app_));
+    ON_CALL(*mock_app_,
+            AreCommandLimitsExceeded(
+                static_cast<mobile_apis::FunctionID::eType>(kFunctionId),
```

Comment:
@okozlovlux, perhaps it would be better to store function id as `FunctionID` enum, rather then `uint32_t`.
In any case, store it in `uint32_t` is bad idea, because function id can hold a `-1` as value.